site/packages/aa-signers/particle/authenticate.md

```diff
@@ -0,0 +1,72 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: ParticleSigner • authenticate
+  - - meta
+    - name: description
+      content: Overview of the authenticate method on ParticleSigner
+  - - meta
+    - property: og:description
+      content: Overview of the authenticate method on ParticleSigner
+---
+
+# authenticate
+
+`authenticate` is a method on the `ParticleSigner` which leverages the `Particle` SDK to authenticate a user.
```

Comment:
It may be worth clarifying here, "to initiate login and authenticate a user" given that `authenticate` directly throws a login modal upon calling